org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/serializer/impl/ChangeSerializer.java

```diff
@@ -80,6 +81,7 @@ else if (context instanceof Resource)
 			else if (context instanceof ResourceSet) {
 				throw new IllegalStateException("Not supported");
 			}
+			progress.worked(1);
```

Comment:
Should we use progress.split(..) instead?

Reply:
Makes sense. Updated, please check again.